nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearchHttpRecord.java

```diff
@@ -669,6 +669,9 @@ private void writeValue(final JsonGenerator generator, final Object value, final
                 }
                 break;
             }
+            case DECIMAL:
```

Comment:
I think this should actually be a double because that's the maximum that Elasticsearch supports. See here:

https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html